monitor.py

```diff
@@ -1,57 +1,39 @@
-import time
-from datetime import datetime
-from temp_humid import get_temperature_and_humidity
+from sensors import get_measurement
 from microclimate_validator import validate_climate
-from db import engine, measurements
+from db import measurements_insert, mean_of_last_n_measurements
 from logger import logger, initialize_logger
 from mail_sender import MailSender, sender, receiver_email
-from statistics import mean
+from timing import MonitorTimer
+from config import Config
 
-message = """\
-Subject: microclimate
 
-Hi, something wrong with your microclimate. It's time to something
-This message is sent from Python.
-Rasberry Pi started monitoring.
-"""
+def main():
+    initialize_logger('pimicroclimate.log')
 
-MONITORED_VALUES = ('temperature', 'humidity')
-mail_sender = MailSender(sender, receiver_email, MONITORED_VALUES)
+    MONITORED_VALUES = ('temperature', 'humidity')
```

Comment:
I think we can leave it global constant somewhere if we plan to re-use it.

Reply:
what about move it to config?